### PR TITLE
Adjust the FCP durations for QM's needs

### PR DESCRIPTION
**Pull request by EnnuiL**
While this also makes the FCPs match the reality, I also propose the following:
 - FCPs for `t: new` and `t: documentation` can be fast-tracked to 1 day if there are 3 approvals *and* it targets a snapshot
 - FCPs for `t: refactor` has been shortened to 2 days due to how FCPs longer than 3 days have been hurting us, and plus the requirement for 2 "core" approvals *and* 3 approvals in general have already inflated the time it takes for such a PR to be approved anyway, with the 2 "core" approvals already making them go through higher scrutiny than usual
   - Update: `t: medium` and `t: large` refactors, however, will respectively have 3 and 4 days of FCP, with the expectation being that the PR indeed is a massive refactoring PR (this ideally means that a one-line refactor in the middle of a `t: new` and `t: medium` PR does not count for bumping the days)
 - FCPs for `t: toolchain` now require 2 approvals, although now that the mappings team is big enough for that to be constantly met, I believe that is the standard nowadays (and plus, toolchain PRs tend to be of the kind that doesn't benefit much from a fast iteration mentality)
 - The Ennui Manuever™ has been ported to Quilt Mappings, allowing the Mappings team to alter individual FCPs if there is a consensus
 
A FCP policy for Quilt Mappings must emphasize the need of fast iteration, especially for a project that needs it, and we can't afford FCPs longer than 2 days at this moment

### CODE REVIEW

**Review by IotaBread — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,58 +72,52 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
...
 
 **Required Approvals**: 3
-- At least 2 approvals must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 2 approvals must come directly from a Mappings team member.
 
-**Final Comment Period**:
-- Snapshots: 3 days
-- Releases: 6 days
+**Final Comment Period**: 2 days
```
Comment (LambdAurora) [inline CONTRIBUTING.md]: I'd add something saying that if the PR is really big the FCP period can be longer as seen fit.

Comment (EnnuiL) [reply CONTRIBUTING.md]: Ah sure, I can introduce a mechanism for that; While it's a bit tempting to rely on the PR being more than `t: medium`, yeah, I might make this like, something to be decided manually

Comment (eristhea) [reply CONTRIBUTING.md]: I would personally bump it up one day for every stage above small: 3 for medium, 4 for large

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,58 +72,52 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
...
 
 ### `T: documentation`
 
 **Description**: Used for pull requests that focus mainly on adding or changing documentation.
 
 **Required Approvals**: 2
-- At least 1 approval must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 1 approval must come directly from a Mappings team member.
```
Comment (eristhea) [inline CONTRIBUTING.md:104]: I don't like having a single point here, I think we have a big enough team that we can still say 2 reviews are necessary

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -72,58 +72,52 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
...
 
 **Final Comment Period**:
-- Snapshots: 2 days
-- Releases: 4 days
+- Snapshots: 2 days (1 day with 3+ approvals)
+- Releases: 2 days
```
Comment (eristhea) [inline CONTRIBUTING.md]: just a note, these can merged since they're identical

File: CONTRIBUTING.md (hunk 2 above)
Comment (EnnuiL) [reply CONTRIBUTING.md:104]: I'm confused here; the intention here is to make it clear that only 1 review from the Mappings team is required while the rest can come from the Mappings Triage team, I didn't change the number of approvals here

File: CONTRIBUTING.md (hunk 3 above)
Comment (EnnuiL) [reply CONTRIBUTING.md]: wdym?

File: CONTRIBUTING.md (hunk 1 above)
Comment (EnnuiL) [reply CONTRIBUTING.md]: ah, I agree with that

File: CONTRIBUTING.md (hunk 2 above)
Comment (eristhea) [reply CONTRIBUTING.md:104]: ah I misinpreted that as eliminating the requirement for a triage review

File: CONTRIBUTING.md (hunk 3 above)
Comment (eristhea) [reply CONTRIBUTING.md]: `2 days (1 day with 3+ approvals for snapshots)`
current format is fine though

**Review by eristhea — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (EnnuiL) [reply CONTRIBUTING.md]: ah, I see

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -72,64 +72,55 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
...
-- At least 2 approvals must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 2 approvals must come directly from a Mappings team member.
 
 **Final Comment Period**:
-- Snapshots: 3 days
-- Releases: 6 days
+- `T: small` and below: 2 days
+- `T: medium`: 3 days
+- `T: large` and above: 2 days
```
Comment (supersaiyansubtlety) [inline CONTRIBUTING.md]:
```suggestion
- `T: large` and above: 4 days
```
I assume this is a typo based on [this](https://github.com/QuiltMC/quilt-mappings/pull/437#discussion_r1210912642) conversation?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (EnnuiL) [reply CONTRIBUTING.md]: oh whoops

**Review by supersaiyansubtlety — APPROVED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -72,64 +72,55 @@ with [RFC 39](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-poli
...
-- At least 2 approvals must come directly from a Mappings team member, and the other one can come from a Mappings
-Triage member or another Mappings Team member.
+- At least 2 approvals must come directly from a Mappings team member.
 
 **Final Comment Period**:
-- Snapshots: 3 days
-- Releases: 6 days
+- `T: small` and below: 2 days
+- `T: medium`: 3 days
+- `T: large` and above: 4 days
```
Comment (eristhea) [inline CONTRIBUTING.md]: these should be `s:` not `t:`

**Review by eristhea — CHANGES_REQUESTED** [review]

**Review by eristhea — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Review by LambdAurora — APPROVED** [review]

**Pull request merged**